core/src/main/java/uk/ac/bbsrc/tgac/miso/core/event/alerter/EmailAlerterService.java

```diff
@@ -60,7 +62,7 @@ public void raiseAlert(Alert a) throws AlertingException {
       throw new AlertingException("No SMTP host specified in the mail.properties configuration file. Cannot send email.");
     } else {
       String from = mailProps.getProperty("mail.from");
-      if (from == null || "".equals(from)) {
+      if (!isStringEmptyOrNull(from)) {
```

Comment:
Ooops. I'll fix this. Also, do we need the empty check at all, or should this just use the default argument to `getProperty`?